cvmfs/cvmfs_config

```diff
@@ -1437,12 +1438,23 @@ reload_service() {
 
 
 configure_autofs() {
+  for path in /etc/auto.master /etc/autofs/auto.master; do
+      if [ -f $path ]; then
+          AUTOMASTER=$path
+          break;
+      fi
+  done
+  if [ -z "$AUTOMASTER" ]; then
+      echo "Error: unable to find auto.master in any of the supplied paths!"
+      num_errors=$(($num_errors+1))
```

Comment:
This is only meant used in the context of chksetup.  At this point, you can just `exit 1`